### ISSUE TITLE
Doesn't run on unix like OS =)

### ISSUE THREAD

**Issue by QuAzI** [Wolfsblvt/BlossomsPokemonGoManager#296]
First problem

``` bash
 ./RunOnUnixLikeSystems.sh 
bash: ./RunOnUnixLikeSystems.sh: Access denied
```

Because before run shell script must have executable attribute.

Fixed by

```
> chmod +x RunOnUnixLikeSystems.sh
```

Second problem

``` bash
> cat RunOnUnixLikeSystems.sh 
#!/usr/bin/env bash
start java -jar BlossomsPogoManager.jar
> ./RunOnUnixLikeSystems.sh 
start: invalid option: -j
Try `start --help' for more information.
```

My script (work from BPGM_v1.1beta)

``` bash
#!/bin/sh
java -jar BlossomsPogoManager.jar &
```

**Comment by Wolfsblvt** [comment]
That's really strange. I mean I do not work on Mac that often, but I really thought `#!/usr/bin/env bash` was the correct shebang line.

Do you know how you can set the executable rights on a file by default on windows or that it get saved with those rights in git?

**Comment by Omeryl** [comment]
@Wolfsblvt it is a correct shebang. He just corrected it for no reason :p.

**Comment by Wolfsblvt** [comment]
So only the file rights have to be fixed?

**Comment by Omeryl** [comment]
@Wolfsblvt and you just need to not use `start` on OS X.

**Comment by Wolfsblvt** [comment]
Ah. So that's windows stuff too.
Then just use `java -jar BlossomsPogoManager.jar`?

**Comment by QuAzI** [comment]
Shebang #!/bin/sh is more classic for unix like OS (as example in FreeBSD by default bash not installed) but real trouble is command which run java.
Unix rights can't be assigned from Windows and not saved in repo, but you can write about it in README. Also why readme extension changed to txt? I think Markdown is more preferred.

**Comment by Wolfsblvt** [comment]
Txt is the readme that gets shipped with the release zip. How could you use cross-platform markdown there? xD
The readme on GitHub will still stay the same.

**Comment by QuAzI** [comment]
Cross-platform software (like Atom) =) 
Markdown is still readable as plain text but Markdown is better for documentation =) 
Senseless always convert md to txt. Just save your time

**Comment by Wolfsblvt** [comment]
Fixed the script with PR #329, added a short note on the readme with #328 for Unix users.
